openassessment/xblock/peer_assessment_mixin.py

```diff
@@ -1,5 +1,6 @@
 import logging
 
+
```

Comment:
Nice work @aliassad to move `get_prompt_templates` in our app, but you've left some empty lines in this core file, would you please remove these empty lines as well?

Reply:
yes removed